lib/components/auth/auth-component.js

```diff
@@ -44,7 +44,7 @@ class AuthComponent extends Component {
           const headers = msg.data.toString().split('\n')
```

Comment:
it seems the line above this one was no longer used (`authHeader = 'Basic' + ...`), can you remove it in this PR too? That would be great.

Reply:
I meant, in another PR (if you have time). I also see that the comment in the component is no longer accurate.